spatial_cell.cpp

```diff
@@ -654,7 +654,7 @@ namespace spatial_cell {
             // Send this data only to ranks that contain face neighbors
             // this->neighbor_number_of_blocks has been initialized to 0, on other ranks it can stay that way.
             const set<int>& ranks = this->face_neighbor_ranks[neighborhood];
-            if ( receiving || ranks.find(receiver_rank) != ranks.end()) {
+            if ( P::amrMaxSpatialRefLevel == 0 || receiving || ranks.find(receiver_rank) != ranks.end()) {
```

Comment:
So this has no alternative path for >0?

Reply:
It does, the alternative path is everything to the right of the first ||. I had to add the == 0 case to make it work when there is no AMR.

Reply:
O'rite.